### PR TITLE
Blockshield

### PR DESCRIPTION
**Pull request by Lamasmaster**
## About The Pull Request
<details>
<summary>
	About The Pull Request
</summary>
<hr>

Splits blackshield from marshals in every way except map, they aren't same organization and haven't been for long time so it should be official, also means no more paying fines with marshal money as one.
On manifest to account for this now show as ''Nadezhda Marshals'' and 'Blackshield Militia' no longer being under same tab.
Their own account has just enough to pay for wages and nothing else.
Tagilla only targets blackshield like it's supposed to.
Now with their own backpacks on spawn too, that someone made not sure who but since they were in files I finished the missing states for them.
	
<hr>
</details>

## Changelog
:cl:
add: Blackshield as it's own department
/:cl:

### CODE REVIEW

File: code/__DEFINES/jobs.dm [hunk 1]
```diff
@@ -30,8 +32,8 @@
 
 #define DEPARTMENT_GREYSON "Greyson Positronic"
 
-#define ALL_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_MEDICAL, DEPARTMENT_ENGINEERING, DEPARTMENT_SCIENCE, DEPARTMENT_SECURITY, DEPARTMENT_LSS, DEPARTMENT_CIVILIAN, DEPARTMENT_CHURCH, DEPARTMENT_PROSPECTOR, DEPARTMENT_INDEPENDENT)
+#define ALL_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_MEDICAL, DEPARTMENT_ENGINEERING, DEPARTMENT_SCIENCE, DEPARTMENT_SECURITY, DEPARTMENT_BLACKSHIELD, DEPARTMENT_LSS, DEPARTMENT_CIVILIAN, DEPARTMENT_CHURCH, DEPARTMENT_PROSPECTOR, DEPARTMENT_INDEPENDENT)
 
 #define ASTER_DEPARTMENTS list(DEPARTMENT_COMMAND, DEPARTMENT_LSS)
 
-#define JOBS_OVERALL list(JOBS_SECURITY, JOBS_COMMAND, JOBS_ENGINEERING, JOBS_MEDICAL, JOBS_SCIENCE,JOBS_LSS,JOBS_CIVILIAN,JOBS_CHURCH,JOBS_PROSPECTOR)
+#define JOBS_OVERALL list(JOBS_COMMAND, JOBS_ENGINEERING, JOBS_MEDICAL, JOBS_SCIENCEm JOBS_SECURITY, JOBS_BLACKSHIELD, JOBS_LSS, JOBS_CIVILIAN, JOBS_CHURCH,JOBS_PROSPECTOR)
```
Comment (ActuallyALovelyGrave) [inline code/__DEFINES/jobs.dm:39]: `JOBS_SCIENCEm`, should be probably `JOBS_SCIENCE,`

Comment (ActuallyALovelyGrave) [reply code/__DEFINES/jobs.dm:39]: Also no space in-between `JOBS_CHURCH,JOBS_PROSPECTOR`

Comment (Lamasmaster) [reply code/__DEFINES/jobs.dm:39]: Typo moment

**Comment by Trilbyspaceclone** [comment]
With this change, marshals and blackshield will count as 2 different departements to be constant with how we connect departmental slots

**Comment by Lamasmaster** [comment]
Truly a *sniff moment

**Comment by cdb-is-not-good** [comment]
> With this change, marshals and blackshield will count as 2 different departements to be constant with how we connect departmental slots

Mmmmmno. That's not how that works, that was a headmin level decision for how the two dept rule works and doesn't get undone just because the code changes a bit.

**Comment by benj8560** [comment]
Oh this returns...well. I'd rather like to point out this is a pretty notable nerf in most cases. A common argument for WO/Spec handing out bullets/guns/paying for things for shield is well exactly this the joint account. Perhaps a SOP defining what kind of gear shield can get for free from marshals disks/bullet fab (when in good supply) would be worth thinking about. Or we just say goodbye to fun for blackshield when they got no command.

**Comment by Lamasmaster** [comment]
You mean how marshals don't get to have any fun either when spec isn't around when half of their weapons require specilized magazines that you don't get on disk (cells, 10mm SMG mags, shotgun tubes). This would realistically change nothing except both get to manage their own budgets, both are still part of security as a whole like soteria medical / research so there is no real reason to scrape off the other side. For a while now too blackshield has been using marshal roundstart disks and their fab in TACARM too which is still a thing I haven't taken away their access to that with teleporter existing so they aren't completely screwed out of gear even when spec is around.

**Comment by Lamasmaster** [comment]
*Sniff mfw this somehow doesn't fix the conflicts

**Comment by Lamasmaster** [comment]
Yeah no there is no real way to unscrew it at this point without a git client so I will simply remake it, probably going to do it way cleaner there then too

**Pull request closed without merging**